packages/Utils/src/DTK_ConfigDefs.hpp

```diff
@@ -18,34 +18,17 @@
 #include "DataTransferKitUtils_config.hpp"
 
 #include <boost/current_function.hpp>
-#include <cstdint>
 #include <string>
 
 namespace DataTransferKit
 {
-//---------------------------------------------------------------------------//
-
-//! Coordinate typedef.
-using Coordinate = double;
-
-//! Local ordinal typedef.
-using LocalOrdinal = unsigned int;
 
-//! Global ordinal typedef.
-using GlobalOrdinal = uint64_t;
+#include "DTK_Types.h"
```

Comment:
your changes make sense here but then I wonder why keeping `DTK_ConfigDefs.hpp`.  The `REGION_NAME` macro was a weird fit in the 1st place...